File: DKPython/functions/dk_debug.py
import sys 
sys.path.append(".")
import DK

from dk_log import *

##################################################################################
# dk_debug(<message>)
#
#    Print a debug message to the console
#
#    @message - The message to print
#
def dk_debug(*args):
#    dk_call dk_debugFunc 1

    dk_log(DEBUG, args[0])





###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
def DKTEST():
#    dk_call dk_debugFunc 0

    dk_debug("test dk_debug message")

#DKTEST()

File: DKPhp/functions/dk_debug.php
<?php

################################################################################
# dk_debug(message)
#
#    Print a debug message to the console
#
#    @message - The message to print
#
function dk_debug() {
	#dk_debugFunc(1);
	dk_source("dk_log");
	
	if(!isset($argv)){
		$argv = func_get_args();
	}
	
    dk_log("DEBUG", $argv[0]);
}




###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
if(!function_exists('DKTEST')){ function DKTEST() {
	#dk_debugFunc 0

    dk_debug("test dk_debug message");
}}


include_once(str_replace("\\","/",$_SERVER['USERPROFILE'])."/DigitalKnob/Development/DKPhp/functions/DK.php");
?>